.github/actions/check-image-and-changes/action.yaml

```diff
@@ -0,0 +1,60 @@
+name: "Check Image and Relevant Changes"
+description: "Checks if the image already exists and if changes are relevant"
+outputs:
+  image_exists:
+    description: "True if the image already exists"
+    value: ${{ steps.image-check.outputs.exists }}
+  relevant_changes:
+    description: "True if changes require a build"
+    value: ${{ steps.changes.outputs.relevant }}
+  short_sha:
+    description: "Short SHA of the latest commit"
+    value: ${{ steps.sha.outputs.short_sha }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Get the last commit short SHA of the PR
+      id: sha
+      shell: bash
+      run: |
+        SHORT_SHA=$(git rev-parse --short HEAD)
```

Comment:
probably better to reuse this action for consistency https://github.com/redhat-developer/rhdh/tree/main/.github/actions/get-sha